server/datastore/mysql/app_configs.go

```diff
@@ -123,6 +123,7 @@ func applyEnrollSecretsDB(ctx context.Context, q sqlx.ExtContext, teamID *uint,
 	}
 	secretsCreatedAt := make(map[string]*time.Time, len(existingSecrets))
 	for _, es := range existingSecrets {
+		es := es
```

Comment:
Did this fix a bug? :)

Reply:
it might! I'm not completely sure. How cool is that new check?

Reply:
This is an awesome check. The issue is that we were taking the address of `es.CreatedAt` in the loop, but `es` gets modified on each loop iteration, so that pointer pointed to the last `CreatedAt` in the `existingSecrets` slice. Unlikely to have been a big issue (since this is just the created-at timestamp), but still...